### PR TITLE
Add `metadata_match` fields and remove cborAuxdataPositions from artifacts

### PR DESCRIPTION
**Pull request by marcocastignoli**
- add `metadata_match`
- remove cborAuxdataPositions from artifacts
- update verifier-alliance test files accordingly
- update tests

### CODE REVIEW

File: services/database/migrations/20240710103541-add-metadata_match.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+exports.up = function (db, callback) {
+  async.series(
+    [
+      db.runSql.bind(
+        db,
+        `ALTER TABLE public.verified_contracts ADD runtime_metadata_match bool NOT NULL;`,
+      ),
+      db.runSql.bind(
+        db,
+        `ALTER TABLE public.verified_contracts ADD creation_metadata_match bool NOT NULL;`,
```
Comment (manuelwedler) [inline services/database/migrations/20240710103541-add-metadata_match.js:28]: Shouldn't it be `ADD COLUMN`?

File: services/database/migrations/20240710103541-add-metadata_match.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+exports.down = function (db, callback) {
+  async.series(
+    [
+      db.runSql.bind(
+        db,
+        `ALTER TABLE sourcify_matches DROP COLUMN runtime_metadata_match;`,
+      ),
+      db.runSql.bind(
+        db,
+        `ALTER TABLE sourcify_matches DROP COLUMN creation_metadata_match;`,
```
Comment (manuelwedler) [inline services/database/migrations/20240710103541-add-metadata_match.js:44]: These queries are altering the wrong table. Should be `public.verified_contracts`.

**Review by manuelwedler — CHANGES_REQUESTED** [review]
The db migrations need some changes. The rest looks good :+1:

File: services/database/migrations/20240710103541-add-metadata_match.js (hunk 1 above)
Comment (marcocastignoli) [reply services/database/migrations/20240710103541-add-metadata_match.js]: apparently it works in both ways :)

Comment (manuelwedler) [reply services/database/migrations/20240710103541-add-metadata_match.js]: ah I didn't know! Being a bit more verbose here makes the query at least more understandable :)

**Review by manuelwedler — APPROVED** [review]

**Pull request merged**